website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -9,6 +9,10 @@ description: >-
 
 Keep track of changes to the API for HCP Terraform and Terraform Enterprise.
 
+## 2024-07-18
+* Add `default-project` relationship to the [`organization` API](/terraform/cloud-docs/api-docs/organizations).
```

Comment:
I see a couple other places in the changelog where we refer to APIs by name, so I would recommend:

```suggestion
* Add `default-project` relationship to the [Organizations API](/terraform/cloud-docs/api-docs/organizations).
```